root/static/scripts/edit/externalLinks.js

```diff
@@ -91,15 +108,15 @@ class ExternalLinksEditor extends React.Component {
   }
 
   getOldLinksHash() {
```

Comment:
Shall we check return type (similarly to parameters type)?

Reply:
Good point actually. Flow has type inference, so normally we don't have to specify types where it can figure them out on its own. But for external modules, [it says this](https://flow.org/en/docs/libdefs/#toc-general-best-practices):

> If a third-party library that has no type information is used by your project, Flow will treat it like any other untyped dependency and mark all of its exports as `any`. Interestingly, this is the only place that Flow will implicitly inject `any` into your program.

Since we're returning the result of a lodash function, I believe the inferred return type is `any`, which disables type checking wherever the function is used.

So adding return types is a good idea wherever we're calling an external library function and returning its result. A better idea, if possible, is pulling in types for the library via `flow-typed` (mentioned in the link above), and letting flow infer the types from that.

Reply:
flow-typed only has types for lodash v4, so I'd prefer to do that in a later PR.

Reply:
I started to tickle updating to lodash v4 alredy.